lib/CounterInput.tsx

```diff
@@ -31,6 +31,7 @@ export interface ICounterInputProps {
   min?: number;
   max?: number;
   borderRadius?: number;
+  reverseButton?: boolean;
```

Comment:
Naming should be different something like `reverseCounterButtons`. I approved this and will change the naming.